[DRAFT/RFC] Documentation expansion (README.md, CONTRIBUTING.md, CONVENTIONS.md) (#1062)
FabricMC/fabric: README.md

# Fabric API

Essential hooks and patches for modding with Fabric.

> Fabric API is the library for essential hooks and interop mechanisms for Fabric mods. Examples include:

Reviewer: ```suggestion
Fabric API is the library for essential hooks and interoperability mechanisms for Fabric mods. Examples include:
```

Full word here is better imo

- Exposing functionality that is useful but difficult to access for many mods such as particles, biomes and dimensions
- Adding events, hooks and APIs to improve interopability between mods.
- Essential features such as registry synchronization and adding information to crash reports.
- An advanced rendering API designed for compatibility with optimization mods and graphics overhaul mods.

Also check out [Fabric Loader](https://github.com/FabricMC/fabric-loader), the (mostly) version independent mod loader that powers Fabric. Fabric API is a mod like any other which requires Fabric Loader to be installed.

For support and discussion for both developers and users, visit [the Fabric Discord server](https://discord.gg/v6v4pMv).

For more information about the Fabric project as a whole check out the [Fabric homepage](https://fabricmc.net/), and the [Fabric wiki](https://fabricmc.net/wiki/doku.php).

## Getting Started

### For Players

> To install Fabric, you need the **Fabric Loader**, which you can download [here](https://fabricmc.net/use).

Reviewer: ```suggestion
To install Fabric, you need to have **Fabric Loader** installed. Downloading and instructions for installing Fabric Loader can be found [here](https://fabricmc.net/use).
```

For more detailed instructions, look [here](https://fabricmc.net/wiki/install).

> In addition, most mods depend on an API mod, containing commonly used hooks. It can be found [here](https://minecraft.curseforge.com/projects/fabric/files).

Reviewer: I'd probably also mention github releases here

> In addition, most mods depend on an API mod, containing commonly used hooks. It can be found [here](https://minecraft.curseforge.com/projects/fabric/files).

Reviewer: ```suggestion
In addition, most mods depend on Fabric API. Fabric API can be downloaded [here](https://minecraft.curseforge.com/projects/fabric/files).
```

### For Developers

To setup a Fabric development environment, check out the [example Fabric mod](https://github.com/FabricMC/fabric-example-mod) and follow the instructions there.

> The Fabric wiki also has a series of helpful tutorials to teach you how to create your first mod. They can be found [here](https://fabricmc.net/wiki/start#tutorials2).

Reviewer: ```suggestion
The [Fabric wiki](https://fabricmc.net/wiki/start#tutorials2) has a series of helpful tutorials to teach you the basics of mod development. The wiki includes tutorials on topics such as registering blocks, items and more advanced topics such as commands.
```

### For Contributors

> If you're looking to contribute to Fabric API take a look at the [contributions page](CONTRIBUTING.md) for information on how to get started.

Reviewer: There is probably a more inviting way to word this.
